src/redux/actions/embedding/loadEmbedding.js

```diff
@@ -9,10 +9,21 @@ const loadEmbedding = (experimentId, embeddingType) => async (dispatch, getState
     return null;
   }
 
-  // Do not allow loading when we are no longer in a loading state if there is no error condition.
-  if (getState().embeddings[embeddingType] && !getState().embeddings[embeddingType].error) {
-    return null;
-  }
```

Comment:
Why did this get removed? My understanding is that we are still going to need to prevent loading on an error condition?

Reply:
The if statement basically says "If everything is ok, and there is no error, then don't allow reload". However, the ticket requires updates to embedding when configuration changes, even when the plot loads correctly.

Removing this allows the plot to be reloaded when the loaded data is ok. Otherwise, the embedding can not be reloaded with new.